cms/templates/howitworks.html

```diff
@@ -39,7 +39,7 @@ <h2 class="sr">${_("{studio_name}'s Many Features").format(studio_name=settings.
             <img src="${static.url("images/thumb-hiw-feature1.png")}" alt="${_('{studio_name} Helps You Keep Your Courses Organized').format(studio_name=settings.STUDIO_SHORT_NAME)}" />
             <figcaption class="sr">${_("{studio_name} Helps You Keep Your Courses Organized").format(studio_name=settings.STUDIO_NAME)}</figcaption>
             <span class="action-zoom">
-              <i class="icon fa fa-search-plus"></i>
+              <span class="icon fa fa-search-plus" aria-hidden="true"></span>
```

Comment:
For each of these, let's add `<span class="sr">Enlarge image</span>` after the icon.

More work needs to be done on this page, but that's for another ticket.